### PR TITLE
Suites labels utility 

### PR DESCRIPTION
**Pull request by epszaw**
### Context

Fixes the problem with deep suites structure for jasmine reporter.
Implements utility to extract and assign suites labels to a given test.
Replaces `ts-node` by `tsx` (no more types definition errors!)

#### Checklist
- [x] [Sign Allure CLA][cla]
- [x] Provide unit tests

[cla]: https://cla-assistant.io/accept/allure-framework/allure2

### CODE REVIEW

File: packages/allure-hermione/test/.hermione.conf.js [hunk 1]
```diff
@@ -9,8 +9,8 @@ module.exports = {
   browsers: {
     headless: {
       desiredCapabilities: {
-        browserName: "chrome",
-        "goog:chromeOptions": {
+        browserName: "firefox",
```
Comment (epszaw) [inline packages/allure-hermione/test/.hermione.conf.js:12]: Did that due `chromedriver` instability. `geckodriver` works much more stable and doesn't have conflicts with locally installed browser and selenium grid one.

**Review by vovsemenv — APPROVED** [review]

File: packages/allure-hermione/src/index.ts [hunk 2]
```diff
@@ -73,7 +74,8 @@ const hermioneAllureReporter = (hermione: Hermione, opts: AllureReportOptions) =
     const thread = ALLURE_THREAD_NAME || test.sessionId;
     const hostnameLabel = ALLURE_HOST_NAME || hostname;
     const currentTest = new AllureTest(runtime, Date.now());
-    const [parentSuite, suite, ...subSuites] = getSuitePath(test);
+    const suites = getSuitePath(test);
+    const [parentSuite, suite, subSuite] = getSuitesLabels(suites);
```
Comment (baev) [inline packages/allure-hermione/src/index.ts:78]: unused?

**Review by baev — APPROVED** [review]

File: packages/allure-hermione/src/index.ts (hunk 2 above)
Comment (epszaw) [reply packages/allure-hermione/src/index.ts]: Removed

**Comment by nagarajanragothaman** [comment]
Hi There, i'm using cucumber js with allure reports and my allure reports suite are coming with scenarios names only and not with folder structure. Is this pull request will close my issue as well? Or its only for jasmine?

**Pull request merged**